src/version.js

```diff
@@ -1,4 +1,4 @@
-const version = '{{versionString}}';
+const version = process.env.VERSION; // eslint-disable-line
```

Comment:
How does this work when being run on the web. Is this one of the modules you built, or a new one that's being brought in?